api/managementpb/haproxy.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package management;
+
+option go_package = "api/managementpb;managementpb";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+import "inventorypb/agents.proto";
+import "inventorypb/services.proto";
+import "managementpb/metrics.proto";
+import "managementpb/service.proto";
+
+// Add HAProxy
+
+message AddHAProxyRequest {
+  // Node identifier on which an external exporter is been running.
+  // runs_on_node_id always should be passed with node_id.
+  // Exactly one of these parameters should be present: node_id, node_name, add_node.
+  string runs_on_node_id = 1;
```

Comment:
let's rename `runs_on_node_id` to `node_id` and remove duplicated `node_id` below

Reply:
and update comments

Reply:
Removed and updated.